homeassistant/components/wirelesstag/sensor.py

```diff
@@ -42,31 +76,35 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     sensors = []
     tags = platform.tags
     for tag in tags.values():
-        for sensor_type in config.get(CONF_MONITORED_CONDITIONS):
-            if sensor_type in tag.allowed_sensor_types:
-                sensors.append(
-                    WirelessTagSensor(platform, tag, sensor_type, hass.config)
-                )
+        for description in SENSOR_TYPES:
+            key = description.key
+            if key in config.get(CONF_MONITORED_CONDITIONS):
+                if key in tag.allowed_sensor_types:
+                    sensors.append(
+                        WirelessTagSensor(platform, tag, description, hass.config)
+                    )
 
     add_entities(sensors, True)
 
 
 class WirelessTagSensor(WirelessTagBaseSensor, SensorEntity):
     """Representation of a Sensor."""
 
-    def __init__(self, api, tag, sensor_type, config):
+    entity_description: SensorEntityDescription
+
+    def __init__(self, api, tag, description, config):
```

Comment:
Why are we passing `hass.config`? It doesn't seem used.

Reply:
ok, agree. leftover from past. removed.